source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -716,6 +716,22 @@ export abstract class CardElement extends CardObject {
     getActionById(_id: string): Action | undefined {
         return undefined;
     }
+    
+    getElementByIdFromAction(id: string): CardElement | undefined {
+        let result = undefined;
+        for (let i = 0; i < this.getActionCount(); i++) {
+            const action = this.getActionAt(i);
+
+            if (action instanceof ShowCardAction) {
+                result = action.card.getElementById(id);
```

Comment:
do we only have the scenario when we need to pull the id of the action `only` when it's `showCardAction`? 
it's solving this particular issue, just want to make sure that we don't have any other potential scenario with other action types when we need to do the same thing:)

Reply:
Yup, showCardAction is the only action that has its own card as of now